libraries/botbuilder-core/botbuilder/core/status_codes.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from enum import IntEnum
+
+
+class StatusCodes(IntEnum):
+    OK = 200
+    SUCCESSFUL_LIMIT = 299
+    BAD_REQUEST = 400
+    UNAUTHORIZED = 401
+    NOT_FOUND = 404
+    METHOD_NOT_ALLOWED = 405
+    CONFLICT = 409
+    UPGRADE_REQUIRED = 426
+    INTERNAL_SERVER_ERROR = 500
+    NOT_IMPLEMENTED = 501
```

Comment:
so you do have NOT_IMPLEMENTED - why not use it? (or alternatively just use the HTTPStatus everywhere.)

Reply:
Yes, this was due to me doing a literal porting of js PR... I might get rid of StatusCodes